src/main/java/komo/fraczek/toukparking/domain/ParkingMeter.java

```diff
@@ -0,0 +1,94 @@
+package komo.fraczek.toukparking.domain;
+
+
+import javax.persistence.*;
+import java.time.Duration;
+import java.time.LocalDateTime;
+import java.time.temporal.ChronoUnit;
+
+@Entity
+public class ParkingMeter {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long Id;
+
+    //    Holds parking ticket code generated for driver
```

Comment:
co dają te komentarze? jeśli już muszą być to czemu to nie są javadoci?

Reply:
Dobre pytanie, zgadzam się, że nie są specjalnie sensowne.